frontend/src/components/HelpTriggerIcon.vue

```diff
@@ -25,9 +24,12 @@ export default defineComponent({
     },
   },
   setup: (props) => {
-    const event = inject("event") as Event;
+    const helpStore = useHelpStore();
     const handleClick = () => {
-      event.emit(EventType.EVENT_HELP, props.id, props.openByDefault);
+      helpStore.$patch({
```

Comment:
Use pinia action instead of `.$patch` might be better.

Reply:
PTAL